apps/root/src/pages/earn/components/all-strategies-table/index.tsx

```diff
@@ -0,0 +1,211 @@
+            <Skeleton variant="text" animation="wave" />
+          </StyledBodySmallSemiboldTypo2>
+        </TableCell>
+        <TableCell>
+          <ContainerBox gap={2} alignItems="center">
+            <Skeleton variant="circular" width={28} height={28} animation="wave" />
+            <StyledBodySmallSemiboldTypo2>
+              <Skeleton variant="text" animation="wave" width="6ch" />
+            </StyledBodySmallSemiboldTypo2>
+          </ContainerBox>
+        </TableCell>
+        <TableCell>
+          <StyledBodySmallSemiboldTypo2>
+            <Skeleton variant="text" animation="wave" />
+          </StyledBodySmallSemiboldTypo2>
+        </TableCell>
+        <TableCell>
+          <ContainerBox>
+            <StyledYieldTypeBox>
+              <StyledBodySmallRegularTypo2>
+                <Skeleton variant="text" animation="wave" width="6ch" />
+              </StyledBodySmallRegularTypo2>
+            </StyledYieldTypeBox>
+          </ContainerBox>
+        </TableCell>
+        <TableCell>
+          <StyledBodySmallSemiboldTypo2>
+            <Skeleton variant="text" animation="wave" />
+          </StyledBodySmallSemiboldTypo2>
+        </TableCell>
+        <TableCell>
+          <StyledBodySmallSemiboldTypo2>
+            <Skeleton variant="text" animation="wave" />
+          </StyledBodySmallSemiboldTypo2>
+        </TableCell>
+      </TableRow>
+    ))}
+  </>
+);
+
+const createRow = (strategy: Strategy) => (
+  <TableRow>
+    <TableCell>
+      <StyledBodySmallSemiboldTypo2>{strategy.farm.name}</StyledBodySmallSemiboldTypo2>
+    </TableCell>
+    <TableCell>
+      <ContainerBox gap={2} alignItems="center">
+        {strategy.asset.icon}
+        <StyledBodySmallSemiboldTypo2>{strategy.asset.symbol}</StyledBodySmallSemiboldTypo2>
+      </ContainerBox>
+    </TableCell>
+    <TableCell>
+      <StyledBodySmallSemiboldTypo2>{strategy.network.name}</StyledBodySmallSemiboldTypo2>
+    </TableCell>
+    <TableCell>
+      <ContainerBox>
+        <StyledYieldTypeBox>
+          <StyledBodySmallRegularTypo2>{strategy.farm.yieldType}</StyledBodySmallRegularTypo2>
+        </StyledYieldTypeBox>
+      </ContainerBox>
+    </TableCell>
+    <TableCell>
+      <StyledBodySmallSemiboldTypo2>${usdFormatter(strategy.farm.tvl)}</StyledBodySmallSemiboldTypo2>
+    </TableCell>
+    <TableCell>
+      <StyledBodySmallSemiboldTypo2>{strategy.rewards[0].apy}%</StyledBodySmallSemiboldTypo2>
+    </TableCell>
+  </TableRow>
+);
+
+const AllStrategiesTable = () => {
+  const [page, setPage] = React.useState(0);
+  const earnService = useEarnService();
+  const { isLoadingAllStrategies, strategies } = useAllStrategies();
+
+  React.useEffect(() => {
+    const fetchStrategies = async () => {
+      await earnService.fetchAllStrategies();
+    };
+    void fetchStrategies();
+  }, []);
+
+  const visibleRows = React.useMemo<Strategy[]>(
+    () => strategies.slice(page * ROWS_PER_PAGE, page * ROWS_PER_PAGE + ROWS_PER_PAGE),
```

Comment:
have you checked with the BE team that the strategies are actually not going to be paginated on the BE?